$OpenBSD: patch-include_smb2_libsmb2_h,v 1.1.1.1 2019/12/31 11:35:55 ajacoutot Exp $

smb2_connect_share_async: don't return ENOMEM if connect fails.

Index: include/smb2/libsmb2.h
--- include/smb2/libsmb2.h.orig
+++ include/smb2/libsmb2.h
@@ -176,7 +176,7 @@ const char *smb2_get_client_guid(struct smb2_context *
  * status can be either of :
  *    0     : Connection was successful. Command_data is NULL.
  *
- *   <0     : Failed to establish the connection. Command_data is NULL.
+ *   -errno : Failed to establish the connection. Command_data is NULL.
  */
 int smb2_connect_async(struct smb2_context *smb2, const char *server,
                        smb2_command_cb cb, void *cb_data);
